Week6/hardwoong/Mission2/src/layout/HomeLayout.tsx

```diff
@@ -0,0 +1,37 @@
+import { Outlet } from "react-router-dom"
+import { useState, useRef } from "react";
+import Navbar from "../components/Navbar"
+import Footer from "../components/Footer.tsx";
+import Sidebar from "../components/Sidebar";
+
+const HomeLayout = () => {
+  const [sidebarOpen, setSidebarOpen] = useState(true);
+  const sidebarRef = useRef(null);
+
+  return (
+    <div className="relative min-h-screen bg-[#121212] text-white flex">
+      <div
+        className={`fixed top-0 left-0 h-full w-64 z-50 transform transition-transform duration-300 ${sidebarOpen ? "translate-x-0" : "-translate-x-full"}`}>
+        <Sidebar ref={sidebarRef} sidebarOpen={sidebarOpen} setSidebarOpen={setSidebarOpen} />
+      </div>
+
+      <div className={`relative ${sidebarOpen ? "ml-56" : "ml-0"} flex flex-col flex-1 min-h-screen transition-all duration-300`}>
+        <div className={`fixed left-0 right-0 top-0 z-40 transition-all duration-300`}>
```

Comment:
현재 이 부분에서 템플릿 리터럴이 불필요하게 사용되고 있습니다. 더 일관되고 깔끔한 코드베이스를 유지하기 위해, 사용되지 않는 경우에는 일반 따옴표로 수정하면 좋을 것 같습니다!

참고 : https://biomejs.dev/linter/rules/no-unused-template-literal/

```suggestion
        <div className="fixed left-0 right-0 top-0 z-40 transition-all duration-300">
```